amundsen_application/api/jira/v0.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from http import HTTPStatus
+from flask import Response, jsonify, make_response, request
+from flask.blueprints import Blueprint
+from flask import current_app as app
+from typing import Any
+
+from amundsen_application.jira.jira_client import JiraClient
+from amundsen_application.api.utils.request_utils import get_query_param
+
+LOGGER = logging.getLogger(__name__)
+
+jira_blueprint = Blueprint('jira', __name__, url_prefix='/api/jira/v0')
+
+
+@jira_blueprint.route('/issues', methods=['GET'])
+def get_jira_issues() -> Response:
+    """
+    Given a table key, returns all JIRA tickets containing that key. Returns an empty array if none exist
+    :return: List of JIRA tickets
+    """
+    try:
+        table_key = get_query_param(request.args, 'key', 'Request requires a key')
+        missing_config_response = validate_jira_properties()
+        if len(missing_config_response) > 0:
+            return make_response(jsonify({'msg': missing_config_response}), HTTPStatus.NOT_IMPLEMENTED)
+
+        jira_client = JiraClient()
+        response = jira_client.search(table_key)
```

Comment:
Does this search method limits the result to 3?

Reply:
yes

Reply:
i'm going to get user feedback to see if this was a good way to go or not

Reply:
Cool. Btw, apart from what number is proper to show, how about making the method to accept the number of results?

Reply:
yeah, i'll make that configurable.